test/C/src/federated/SmallDelayDecentralized.lf

```diff
@@ -44,13 +44,18 @@ reactor Print {
     }
     self->c++;
   =} STP(1 sec) {=
-    lf_print_error_and_exit("STP violation at tag " PRINTF_TAG
-        ". This should not happen because the STP offset is large.",
+    lf_print_warning("STP violation at tag " PRINTF_TAG
```

Comment:
This ensures that the test never fails, but the warning will never be shown when the test it executed in our testing framework. So what's the point of still having this test?

Reply:
Ah, there is indeed another way the test can fail, which I hadn't noticed. Thanks for the fix!